src/govuk/components/pagination/_index.scss

```diff
@@ -0,0 +1,170 @@
+@include govuk-exports("govuk/component/pagination") {
+  .govuk-pagination {
+    border-top: 1px solid $govuk-border-colour;
+    text-align: center;
+
+    @include govuk-media-query($from: tablet) {
+
+      // Hide whitespace between elements
+      font-size: 0;
+
+      // Trick to remove the need for floats
+      text-align: justify;
+
+      &:after {
+        content: '';
+        display: inline-block;
+        width: 100%;
+      }
+    }
+  }
+
+  .govuk-pagination__list {
+    margin: 0;
+    padding: 0;
+    list-style: none;
+    @include govuk-media-query($from: tablet) {
+      display: inline-block;
+      margin-bottom: 0;
+    }
+  }
+
+  .govuk-pagination__results {
+    @include govuk-font(19);
+    margin-top: 0;
+    margin-bottom: govuk-spacing(4);
+    padding: govuk-spacing(1) 0;
+    @include govuk-media-query($from: tablet) {
+      display: inline-block;
+      margin-bottom: 0;
+      //vertical-align: middle;
+    }
+  }
+
+  .govuk-pagination__item {
+    @include govuk-font(19);
+    display: inline-block;
+    margin-bottom: govuk-spacing(4);
```

Comment:
Could we move the bottom margin up to `.govuk-pagination__list`? I believe that would mostly work the same, but it would remove what I think of as an awkward gap when the items are stacked:

| Before | After |
| ----- | ----- |
| ![image](https://user-images.githubusercontent.com/100852/143217114-efeab191-8d3b-4df4-bb7c-1b9989d7d53f.png) | ![image](https://user-images.githubusercontent.com/100852/143217148-67fe49d3-9b9f-46e3-9da7-16deba7155dd.png) |

Reply:
Hey Greg - I'd ha a similar thought, but at the moment that's how it works on GOV.UK guidance pages, and I wanted this first iteration to track that design as closely as possible.